app/src/main/java/com/sdex/activityrunner/commons/BaseActivity.kt

```diff
@@ -32,4 +33,12 @@ open class BaseActivity : AppCompatActivity() {
     protected fun setSubtitle(subtitle: CharSequence) {
         supportActionBar?.subtitle = subtitle
     }
+
+    protected fun hideToolbar() {
```

Comment:
Please move these methods to the `ManifestViewerActivity` and use `isVisible = false`